### PR TITLE
Don't show window selector when dragging a systray icon

### PR DESCRIPTION
**Pull request by pushqrdx**

### CODE REVIEW

File: src/taskbar.cpp [hunk 1]
```diff
@@ -750,6 +750,7 @@ pinned_icon_drag_start(AppClient *client_entity, cairo_t *cr, Container *contain
     data->initial_mouse_click_before_drag_offset_x =
             container->real_bounds.x - client_entity->mouse_initial_x;
     container->z_index = 1;
+    possibly_close(app, container, data);
```
[the remark this answers is not in the document]
Comment (pushqrdx) [reply src/taskbar.cpp:753]: @jmanc3 the reason i added it was that for some reason, but ofc it doesn't make sense to add this if it's supposed to close. gonna revert it locally and test again to make sure

[the remark this answers is not in the document]
Comment (pushqrdx) [reply src/taskbar.cpp:753]: @jmanc3 so here's the bug reproduced on master, for some reason the window selector doesn't close if you start drag just while they are about to display or something like that

bug

[the remark this answers is not in the document]
Comment (jmanc3) [reply src/taskbar.cpp:753]: Ah, you're right. Can you add "possibly_close(app, container, data);" to the function "pinned_icon_drag" as well? I'll merge after that!

[the remark this answers is not in the document]
Comment (pushqrdx) [reply src/taskbar.cpp:753]: @jmanc3 added

**Pull request merged**